protelis/protelis-interpreter/src/main/java/org/protelis/lang/ProtelisLoader.java

```diff
@@ -167,10 +185,7 @@ private ProtelisLoader() {
      * @throws IllegalArgumentException
      *             when the program has errors
      */
-    public static ProtelisProgram parse(final String program) {
-        if (Objects.requireNonNull(program, "null is not a valid Protelis program, not a valid Protelis module").isEmpty()) {
-            throw new IllegalArgumentException("The empty string is not a valid program, nor a valid module name");
```

Comment:
Why are we losing this exception information?

Reply:
I see that null is checked for at line 188, but I concur with Jake we are losing information on empty strings. @KaikiKM, can you fix this?

Reply:
Sorry i probably removed it by mistake, fixed